src/isomorphic/React.js

```diff
@@ -28,6 +28,13 @@ var createFactory = ReactElement.createFactory;
 var cloneElement = ReactElement.cloneElement;
 
 if (__DEV__) {
+  console.log(
```

Comment:
Another distinction is that `warning` uses `console.error()`, rather than `console.log()`:

(cf. https://github.com/BerkeleyTrue/warning/blob/master/warning.js#L38)

However IMO `console.warn()` would be the most appropriate in this case.